cypress/e2e/room_board/createEditDeleteFolderInRoomBoard.feature

```diff
@@ -2,32 +2,40 @@
 @stable_test
 Feature: Room Board - Create, edit and delete folder in board, including file handling inside the folder (in progress)
 
-As a teacher I want to create a file folder in a room board
+As a editor I want to create a file folder in a room board
```

Comment:
Editor will be confused easily with RoomEditor. better would be either teacher or user or something similar

Reply:
In our AC they speak about editor, so I want to follow that to be flexible to add examples for other roles who can edit in future.
I would suggest to name it contenteditor, would that be ok?